src/main/java/org/jabref/logic/exporter/ModsExporter.java

```diff
@@ -76,267 +49,343 @@ public void export(final BibDatabaseContext databaseContext, final Path file, Li
         }
 
         try {
-            ModsCollectionDefinition modsCollection = new ModsCollectionDefinition();
+            // create writer
+            IndentingXMLStreamWriter writer = createWriter(file);
```

Comment:
I think, the writer should be in the "try" statement - to have it auto closable:

    try (IndentingXMLStreamWriter writer = createWriter(file))

Reply:
Does not work! StreamWriter does not support auto closable.